src/Auth0UserUpdateUtilities.js

```diff
@@ -0,0 +1,93 @@
+const ManagementClient = require("auth0").ManagementClient;
+
+class Auth0UserUpdateUtilities {
+  constructor(user, auth0, namespace) {
+    this.apiClient = new ManagementClient({
```

Comment:
👍  Glad you were able to find a way to use the newer client.

Reply:
Yeah, it's not perfect, it's just 2 additional attempts 1 second apart (retry logic is not excellent in the SDK) but better than nothing. With this under our control, we can handle this a little smarter in the future.